### PR TITLE
Add login component

### PR DESCRIPTION
**Pull request by kper**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
-    "build-prod": "ng build --base-href /browser/",
```
Comment (luk3k) [inline frontend/package.json:8]: why did you remove the /browser base-href? I think this is needed when running the nginx container (see Dockerfile)

File: docker-compose.yml [hunk 2]
```diff
@@ -67,16 +68,17 @@ services:
       POSTGRES_USER: "postgres"
       POSTGRES_PASSWORD: "postgres"
       POSTGRES_DB: "directus"
+    ports:
+      - "5432:5432"
+    volumes:
+      - ~/volume/db:/var/lib/postgresql/data
```
[the remark this answers is not in the document]
Comment (kper) [reply docker-compose.yml]: Thanks that was a left over of a test

File: frontend/package.json (hunk 1 above)
Comment (kper) [reply frontend/package.json]: I didn't know that the nginx container requires it. I reverted it.

**Comment by Dreistein** [comment]
Looks good from my side 🚀
